### DISCUSSION TITLE
这个问题这么解决啊，老哥。

### DISCUSSION THREAD

**Discussion by Y-5418** [wangxinleo/wechat-public-account-push#36] [General]
![image](https://user-images.githubusercontent.com/74748919/187108400-de1f3cc4-bfac-4444-9cb0-23c2fe4e09d6.png)

**Comment by wangxinleo** [comment]
能不能发一下你的配置图。

**Reply by Y-5418** [reply]
![image](https://user-images.githubusercontent.com/74748919/187109763-e15d205b-4960-480a-b49b-264637511630.png)

**Reply by wangxinleo** [reply]
我好像没看出什么毛病，你用的是测试号吗？

**Reply by LordonCN** [reply]
再run一遍试试，昨晚我也出现过执行失败的情况

**Comment by LordonCN** [comment]
我的定时任务上午也遇到了这个问题
` errmsg: 'invalid credential, access_token is invalid or not latest rid: 630c14a3-566e11b2-77784589' `

**Comment by LordonCN** [comment]
解决了，run 的时候注意选中你修改过的分支，默认是master，会报这个错误
<img width="343" alt="Screen Shot 2022-08-29 at 12 01 10" src="https://user-images.githubusercontent.com/46736284/187120512-070033a1-390e-4a9c-a9cd-9026f8c1e060.png">

**Reply by 20011205** [reply]
怎么选择 branch呢

**Reply by LordonCN** [reply]
你点Branch:master 旁边的箭头就能选。我这边不是直接在master上开发的，所以如果没选对分支就会报这个错误

**Reply by 20011205** [reply]
![哈哈哈](https://user-images.githubusercontent.com/112371755/187135596-22949a21-5605-445c-8901-6c52e8291be6.png)
 只有一个master   没得选

**Comment by wangxinleo** [comment]
@20011205 

0. fork最新仓库
1. 确认自己的号是不是测试号，目前只有服务号才有推送功能，服务号只能企业申请
2. 以下参数对应上了吗？

![1661756547(1)](https://user-images.githubusercontent.com/28293988/187142684-683277a6-5ce9-4d0e-aad4-428e4633cc53.jpg)

![image](https://user-images.githubusercontent.com/28293988/187142816-2f0f6dc0-20ae-48b0-8991-aa3143fbd44f.png)

![image](https://user-images.githubusercontent.com/28293988/187142960-da217045-3859-4179-a40c-e15baeea27c4.png)
